projects/github-actions/test-results-to-slack/src/rules.js

```diff
@@ -0,0 +1,62 @@
+const { getInput } = require( '@actions/core' );
+const debug = require( './debug' );
+const extras = require( './extra-context' );
+
+/**
+ * Returns a list o Slack channel ids, based on context and rules configuration.
+ *
+ * @returns {string[]} an array of channels ids
+ */
+function getChannels() {
+	const channels = [];
+	const defaultChannel = getInput( 'slack_channel' );
+	const rulesConfigurationPath = getInput( 'rules_configuration_path' );
+	const suiteName = getInput( 'suite_name' );
+
+	// If no rules are configured we only use the default channel
+	if ( ! rulesConfigurationPath ) {
+		debug( 'No rules configuration found, returning only the default channel' );
+		channels.push( defaultChannel );
+	}
+
+	if ( rulesConfigurationPath ) {
```

Comment:
Can we combine this conditional with the above and use the `else` case?

Reply:
Yeah, that makes sense. I wonder why I didn't do that in the first place 🤔 . Changed with [8afe2cf](https://github.com/Automattic/jetpack/pull/25830/commits/8afe2cf8a472d9010e15b34d141f3d1a0bb5c96c)